web/src/locust/locustfile.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/python
+from locust import HttpLocust, TaskSet, task
+import json
+import datetime
+
+def get_json_header():
+	return {"content-type":"application/json"}
+
+def get_epoch_timestamp():
+	delta = datetime.datetime.now() - datetime.datetime(1970, 1, 1)
+	return (long(delta.total_seconds()) * 1000) + ((delta.microseconds) / 1000)
```

Comment:
I run python 2.7 on my machine. It is a requirement for locust... if you read the readme ;)